amoro-ams/amoro-ams-server/src/main/java/org/apache/amoro/server/manager/FlinkOptimizerContainer.java

```diff
@@ -532,7 +563,7 @@ protected JobID runJar(String jarId, Configuration configuration, Resource resou
     JobID jobID = JobID.generate();
     JarRunRequestBody runRequestBody =
         new JarRunRequestBody(
-            FLINK_JOB_MAIN_CLASS, args, null, null, jobID, true, null, RestoreMode.DEFAULT, null);
+            FLINK_JOB_MAIN_CLASS, args, null, null, jobID, true, null, RestoreMode.CLAIM, null);
```

Comment:
Do we need to use RestoreMode.CLAIM? Essentially, Optimizer does not checkpoint, the default mode is NO_CLAIM